src/app/features/plants/plantExtended.tsx

```diff
@@ -1,36 +1,77 @@
 import dayjs from "dayjs";
 import { PlantSchema } from "@/app/lib/schema";
-import { Card, CardContent } from "@/components/ui/card";
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from "@/components/ui/collapsible";
+import { Button } from "@/components/ui/button";
+import { ArrowsDown } from "@/components/ui/icons";
 
 interface PlantProps {
   plant: PlantSchema;
 }
 
 export function PlantExtended({ plant }: PlantProps) {
   return (
-    <Card className="border-0">
-      <CardContent>
-        <div className="border">
-          <p className="font-semibold">Instructions</p>
-          <p>{plant.instructions}</p>
+    <Collapsible>
+      <CollapsibleTrigger>
+        <div className="flex gap-1">
+          <ArrowsDown /> Details
         </div>
-        <div className="border">
-          <p className="font-semibold">Volume of water</p>
-          <p>{plant.volume}</p>
+      </CollapsibleTrigger>
+      <div className="flex flex-col gap-2 p-3">
+        <div className="p-2">
+          <div className="">
```

Comment:
why do you need this div to be outside of `CollapsibleContent` and not inside?

Reply:
Thanks for spotting a copied typo :)